pyston/utils/__init__.py

```diff
@@ -320,19 +330,6 @@ def __add__(self, rest_fieldset):
         a_rfs = deepcopy(self)
         return a_rfs.join(rest_fieldset)
 
-    def __sub__(self, rest_fieldset):
-        if isinstance(rest_fieldset, (list, tuple, set)):
-            rest_fieldset = RFS(*rest_fieldset)
-
-        assert isinstance(rest_fieldset, RESTFieldset)
-
-        values = []
-        for rf in self.fields:
-            if rf.name not in rest_fieldset.fields_map:
-                values.append(deepcopy(rf))
-
-        return self.__class__(*values)
```

Comment:
k ničemu to nebylo a navíc to je podle mě špatně naimplementované tak pryč s tím